go/vt/vtorc/logic/tablet_discovery.go

```diff
@@ -64,16 +65,24 @@ func RegisterFlags(fs *pflag.FlagSet) {
 // OpenTabletDiscovery opens the vitess topo if enables and returns a ticker
 // channel for polling.
 func OpenTabletDiscovery() <-chan time.Time {
-	// TODO(sougou): If there's a shutdown signal, we have to close the topo.
```

Comment:
I fixed this too because it was really just 1 line fix, involving adding `ts.Close()` to `closeVTOrc`